yarn.lock

```diff
@@ -4491,7 +4491,7 @@
     ethers "4.0.45"
     eventemitter3 "4.0.0"
 
-"@statechannels/channel-provider@*", "@statechannels/channel-provider@0.0.4":
+"@statechannels/channel-provider@*":
```

Comment:
should this line disappear entirely? AFAICT we don't want any reference to 0.0.4 (see next line)